module/core/src/Application/Normalizer/JMSNormalizer.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Core\Application\Normalizer;
+
+use Ergonode\Core\Infrastructure\Exception\DenoralizationException;
+use Ergonode\Core\Infrastructure\Exception\NormalizerException;
+use JMS\Serializer\ArrayTransformerInterface;
+use JMS\Serializer\DeserializationContext;
+use JMS\Serializer\SerializationContext;
+use Psr\Log\LoggerInterface;
+
+class JMSNormalizer implements NormalizerInterface
```

Comment:
In my opinion, JMSSerializer should implement the interface and have the below implementation.